clients/admin-ui/src/features/data-discovery-and-detection/action-center/fields/MonitorTree.tsx

```diff
@@ -586,20 +573,23 @@ const MonitorTree = forwardRef<MonitorTreeRef, MonitorTreeProps>(
               node={node}
               treeData={treeData}
               onLoadMore={onLoadMore}
+              actions={new Map([...nodeActions.entries()])}
```

Comment:
Isn't nodeActions already a map, why the expensive recreation?
```suggestion
              actions={nodeActions}
```